src/commands/acceptorder.go

```diff
@@ -0,0 +1,156 @@
+package commands
+
+import (
+	"github.com/bwmarrin/discordgo"
+	"sandwich-delivery/src/config"
+	"sandwich-delivery/src/database"
+	"sandwich-delivery/src/models"
+	"strconv"
+	"time"
+)
+
+type AcceptOrderCommand struct{}
+
+func (c AcceptOrderCommand) getName() string {
+	return "acceptorder"
+}
+
+func (c AcceptOrderCommand) getCommandData() *discordgo.ApplicationCommand {
+	return &discordgo.ApplicationCommand{
+		Name:        c.getName(),
+		Description: "Accepts orders.",
```

Comment:
description is a bit lacking to imo

"Accepts the order and assigns it to you."